src/jit/codegencommon.cpp

```diff
@@ -11354,7 +11378,7 @@ instruction CodeGen::genMapShiftInsToShiftByConstantIns(instruction ins, int shi
 
 #endif // _TARGET_XARCH_
 
-#if !defined(LEGACY_BACKEND) && (defined(_TARGET_XARCH_) || defined(_TARGET_ARM64_))
+#if !defined(LEGACY_BACKEND) && (defined(_TARGET_XARCH_) || defined(_TARGET_ARMARCH_))
```

Comment:
This can just be
```
#if !defined(LEGACY_BACKEND)
```
now, since it's available to all current architectures.

Reply:
OK. I'll modify it.